doc/nrf/documentation/build.rst

```diff
@@ -27,6 +27,35 @@ Complete the following steps to install the required tools:
 
 1. If you have not done so already, install the |NCS| as described in :ref:`auto_installation` or :ref:`manual_installation`.
 #. Install or update all required :ref:`Python dependencies <additional_deps>`.
+#. Install the additional Python dependencies for building the documentation.
+   `Python virtual environment <Python virtual environments>`_ needs to be active for the following command to work:
+
+   .. tabs::
+
+     .. group-tab:: Windows
+
+        Enter the following command in a terminal window in the :file:`ncs` folder:
+
+        .. code-block:: bash
+
+           pip install -U -r nrf/doc/requirements.txt
+
+     .. group-tab:: Linux
+
+        Enter the following command in a terminal window in the :file:`ncs` folder:
+
+        .. code-block:: bash
+
+           pip install -U -r nrf/doc/requirements.txt
+
+     .. group-tab:: macOS
+
+        Enter the following command in a terminal window in the :file:`ncs` folder:
+
+        .. code-block:: bash
+
+           pip install -U -r nrf/doc/requirements.txt
```

Comment:
are tabs needed now? all commands are the same

Reply:
I would say we can skip tabs, just that we might have to say that the command is the same for all the three operating systems. @wiba-nordic?

Reply:
Now that the command is the exact same for all three, yeah, the tabs aren't needed anymore. But agreed, clarifying that the command works for Windows, macOS and Linus makes sense.